plugins/homepage/web_client/views/FrontPageView.js

```diff
@@ -10,7 +10,23 @@ wrap(FrontPageView, 'render', function (render) {
         method: 'GET',
         url: 'homepage/markdown'
     }).done(_.bind(function (resp) {
-        this.$el.html(renderMarkdown(resp['homepage.markdown']));
+        if (resp['homepage.markdown'] === '') {
```

Comment:
I think it's fine to just check truthiness in all of these cases. If for some reason the value is `null` or `undefined`, we want to ensure the behavior stays the same.